contrib_bots/bots/encrypt_bot/tests.py

```diff
@@ -0,0 +1,32 @@
+import encrypt_bot
```

Comment:
It doesn't matter too much in this case, but maybe use `from .encrypt_bot import EncryptHandler` to make the same directory import more explicit.